### ISSUE TITLE
BedFiles from MACS2 called peaks have only 10 columns instead of 13

### ISSUE THREAD

**Issue by psreeja** [MarioPujato/NextGenAligner#9]
We used -c option to call peaks using MACS2 and noticed there are only 10 columns in the bedfile output files instead of 13 as mentioned in the README file.

Thanks,

**Comment by MarioPujato** [comment]
Yes, that's because I removed the part where I calculate additional
information on the BED file and, of course, I forgot to remove that part
from the readme.
I am now calculating reads under peaks separately (with the gtfFromPeaks
program).



-------------------------------------------------------------------------
*Mario A. Pujato, Ph.D.*

Operations Director
Bioinformatics Collaborative Services (BCS)
Division of Biomedical Informatics (BMI)
Cincinnati Children's Hospital Medical Center

**Comment by psreeja** [comment]
Ok. Sounds good. Thanks!
